modules/optimization/test/tests/optimizationreporter/function_misfit/main_auto.i

```diff
@@ -0,0 +1,75 @@
+measurement_points = '0.5 0.28 0
+   0.5 0.6 0
+   0.5 0.8 0
+   0.5 1.1 0'
+measurement_values = '293 304 315 320'
+
+[Optimization]
+[]
+
+[OptimizationReporter]
+  type = GeneralOptimization
+  objective_name = objective_value
+  parameter_names = 'parameter_results'
+  num_values = '3'
+[]
+
+[Reporters]
+  [main]
+    type = OptimizationData
+    measurement_points = ${measurement_points}
+    measurement_values = ${measurement_values}
+  []
+[]
+
+[Executioner]
+  type = Optimize
+  tao_solver = taobqnktr
+  petsc_options_iname = '-tao_gatol'
+  petsc_options_value = '1e-8'
+  ## THESE OPTIONS ARE FOR TESTING THE ADJOINT GRADIENT
+  # petsc_options_iname = '-tao_max_it -tao_fd_test -tao_test_gradient -tao_fd_gradient -tao_fd_delta -tao_gatol'
+  # petsc_options_value = '1 true true false 1e-3 1e-5'
+  # petsc_options = '-tao_test_gradient_view'
```

Comment:
```suggestion
```

Reply:
We shouldn't put these in these commented out tao options for the fd gradient in new input files.  I do find them helpful.